### DISCUSSION TITLE
GameSense.is.crack.carck

### DISCUSSION THREAD

**Discussion by h4xrOx** [h4xrOx/GameSense.is.crack.CR4cKCRACK#2] [Announcements]
Crack Crack Crack Cracked of crack and hack. GameSense.is 
[gamesense.crack.crack.zip](https://github.com/h4xrOx/GameSense.is.crack.CR4cKCRACK/files/6832838/gamesense.crack.crack.zip)

<hr /><em>This discussion was created from the release <a href='https://github.com/h4xrOx/GameSense.is.crack.CR4cKCRACK/releases/tag/'>GameSense.is.crack.carck</a>.</em>

**Comment by apprit** [comment]
not working :p showing Hardware Dismatched : A Windows PDB server error has been occurred. Please, contact support. : Error 1377339077 | listen::recv

**Reply by arynaya** [reply]
fr

**Reply by dulp2019** [reply]
no work

**Reply by h4xrOx** [reply]
updated

**Comment by h4xrOx** [comment]
Oh it works, if your a member of h4xr0x.cc

**Reply by apprit** [reply]
u mean the fourms member? i am already but how it works?

**Reply by Broshan1337** [reply]
im getting an error on the forum with some kind of sql error, and i cannot access the forum

**Reply by arynaya** [reply]
did anyone find out how to use it

**Comment by h4xrOx** [comment]
updated

**Comment by h4xrOx** [comment]
updated https://gamesense-pub.azurewebsites.net/ live deployment demo for members; ready to deploy yourself in self build

**Reply by julien112800** [reply]
Error 1377339077 | ::connect wsa_last_error[10060].

on launcher.exe
